src/lib/rest/OrionError.cpp

```diff
@@ -79,7 +79,7 @@ OrionError::OrionError(StatusCode& sc)
 void OrionError::fill(HttpStatusCode _code, const std::string& _details, const std::string& _reasonPhrase)
 {
   code          = _code;
-  reasonPhrase  = _reasonPhrase != ""? _reasonPhrase : httpStatusCodeString(code);
+  reasonPhrase  = !_reasonPhrase.empty()? _reasonPhrase : httpStatusCodeString(code);
```

Comment:
```suggestion
  reasonPhrase  = _reasonPhrase.empty()? httpStatusCodeString(code) : _reasonPhrase;
```

I think is clerer this way, avoiding negation in the condition and swapping the results.

Reply:
Updated